src/main/groovy/com/github/benmanes/gradle/versions/updates/DependencyUpdatesTask.groovy

```diff
@@ -54,11 +58,14 @@ class DependencyUpdatesTask extends DefaultTask {
   @Input
   boolean checkForGradleUpdate = true
 
+  @Input @Optional
   Object outputFormatter = 'plain'
 
+  @Input @Optional
```

Comment:
This is causing the fail in the test, because the closure is executed without correct arguments and delegates during validation. Something similar seems to have happened in https://github.com/ben-manes/gradle-versions-plugin/commit/3cce551e9ced67e38dc99022018f209fd103bea1, where they removed the annotation from the `outputFormatter` field.